fedimint-server/src/config/mod.rs

```diff
@@ -748,6 +758,7 @@ impl ServerConfigParams {
                 let bind_api = parse_host_port(peer_params[peer].clone().api_url)?;
 
                 let params: ServerConfigParams = Self::gen_params(
+                    ApiAuth("dummy_password".to_string()),
```

Comment:
Does this need to be passed from the command line?

Reply:
Just for the CI tests so it's okay to have it as a dummy string for now.

Reply:
It's a little weird to have a hard-coded password like this nonetheless ... wouldn't want anyone to unintentionally use this on mainnet someday after we've forgotten about it ...

Reply:
True, this function should be refactored into the test fixtures since that is the only place it's used.